go/vt/topo/vschema.go

```diff
@@ -17,27 +17,20 @@ limitations under the License.
 package topo
 
 import (
+	"context"
 	"path"
 
 	"google.golang.org/protobuf/proto"
 
-	"context"
-
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/vterrors"
 
 	vschemapb "vitess.io/vitess/go/vt/proto/vschema"
-	"vitess.io/vitess/go/vt/vtgate/vindexes"
 )
 
 // SaveVSchema first validates the VSchema, then saves it.
 // If the VSchema is empty, just remove it.
 func (ts *Server) SaveVSchema(ctx context.Context, keyspace string, vschema *vschemapb.Keyspace) error {
-	_, err := vindexes.BuildKeyspace(vschema)
```

Comment:
You could argue that moving this out makes it more likely that people might forget to call this when writing new code. The counter argument to that is that any functions that call `SaveVSchema` are already calling a complicated sequence of functions to achieve their goals. This just adds one more step that people have to understand, which is fine.